script.js

```
@@ -0,0 +1,76 @@
+let currentQuestion = 0;
+let score = 0;
+let questions = [];
+
+const questionEl = document.getElementById("question");
+const answerButtons = document.querySelectorAll("#answers button");
+const scoreEl = document.getElementById("score");
+
+fetch("data/questions.json")
+.then(response => response.json())
+.then(data => {
+	questions = data.questions;
```

maybe throw an error if questions is empty